setup.py

```diff
@@ -22,23 +21,28 @@
 dependencies = ["sqlparse >= 0.3.0", "google-cloud-spanner >= 1.8.0"]
 extras = {}
 
+BASE_DIR = os.path.dirname(__file__)
+VERSION_FILENAME = os.path.join(BASE_DIR, "version.py")
+PACKAGE_INFO = {}
+with open(VERSION_FILENAME) as f:
+    exec(f.read(), PACKAGE_INFO)
+version = PACKAGE_INFO["__version__"]
```

Comment:
This redeclares the definition in L#15 which thereby seems to have no use.

Reply:
Good catch, forgot to remove L15.